release/release_tests.yaml

```diff
@@ -5107,6 +5107,8 @@
         cluster_compute: stress_tests/stress_tests_compute_gce.yaml
       smoke_test:
         frequency: manual
+    - __suffix__: aws_pa
+      env: aws_pa
```

Comment:
what does the `aws_pa` thing mean?

Reply:
pa: product autoscaler

Reply:
I am not sure if this should be another "env", or maybe should be something clearer, but will leave this to @can-anyscale to judge. :)

Reply:
i guess we just need another product project - which is coded as part of the env. 

This is also more of an intermediate state - we will eventally move everything to "aws_pa" (aws -> aws_pa)